pybossa/cache/__init__.py

```diff
@@ -38,8 +38,10 @@
 
 try:
     from app_settings import config as settings
+    REDIS_KEYPREFIX = settings['REDIS_KEYPREFIX']
 except ImportError:  # pragma: no cover
     import pybossa.default_settings as settings
+    REDIS_KEYPREFIX = settings.REDIS_KEYPREFIX
```

Comment:
It is accessed as `settings.REDIS_KEYPREFIX` in the code below. Don't you need to change that code too?

Reply:
yes, changed it, thank you